CODEOWNERS

```diff
@@ -312,7 +312,7 @@ homeassistant/components/plaato/* @JohNan
 homeassistant/components/plant/* @ChristianKuehnel
 homeassistant/components/plex/* @jjlawren
 homeassistant/components/plugwise/* @CoMPaTech @bouwew
-homeassistant/components/plum_lightpad/* @ColinHarrington
+homeassistant/components/plum_lightpad/* @ColinHarrington @prystupa
```

Comment:
@prystupa Did you actually want to be a codeowner for this platform?  I definitely appreciate the help and you are contributing to adding config-flow to this platform, but are you signing up to be a codeowner of this integration?

Reply:
yes, I don't mind

Reply:
> Looks great. Ok to merge when last comments addressed!

Thumbs up from me: Thank you @prystupa!